docs/contribute.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contribute to Waku
+---
+
+Get involved in Waku's open-source initiatives to improve the protocol, SDKs, developer tools, and documentation.
+
+## Build a Waku Application
+
+By integrating Waku into your application, you can contribute by:
+
+- [Running a node](https://github.com/waku-org/nwaku/tree/master/docs/operators) within your application.
+- Reporting bugs and suggesting missing features to the development team.
+- Inspiring other developers to explore and utilize Waku for [various use cases](/overview/use-cases).
+
+Begin your journey by exploring the [Waku platforms](/platform/overview) and start building today!
+
+## Report a Bug
+
+To report a bug, create an issue in the appropriate [GitHub repository](https://github.com/waku-org). Ensure no issue exists about the bug and include detailed steps to reproduce the bug.
+
+## Request a Feature
+
+To request a new feature, create an issue in the appropriate [GitHub repository](https://github.com/waku-org). Ensure no issue exists about the feature and specify the use cases the feature can enable, allowing us to investigate and prioritize accordingly.
+
+## Make Pull Requests
+
+Community pull requests (PRs) are highly encouraged, but we recommend beginning with a feature request or posting on the [Vac Forum](https://forum.vac.dev/) to gauge interest and gather feedback before proceeding with a PR.
+
+## Contribute to Waku Research
+
+[Vac](https://vac.dev/) drives Waku research, advancing through ongoing R&D. Vac is dedicated to developing modular p2p protocols for private, secure, and censorship-resistant communication. Explore Vac's open problems at <https://vac.dev/open-problems>.
```

Comment:
Waku and Vac are now collaborating.

@jm-clius What would you want to see in the doc here?

Reply:
Since we don't yet have an open problems page for Waku Research, I'd point people to our github repo: https://github.com/waku-org/research. The first part of the section can remove references to Vac and just mention that Waku Research is an open R&D project, developing modular...etc. etc.